packages/core/src/Stores/traders-hub-store.js

```diff
@@ -386,15 +386,15 @@ export default class TradersHubStore extends BaseStore {
         const all_available_accounts = [
             ...getCFDAvailableAccount(),
             {
-                name: !this.is_eu_user || this.is_demo_low_risk ? localize('Financial') : localize('CFDs'),
+                name: !this.is_eu_user || this.is_demo_low_risk ? 'Financial' : 'CFDs',
```

Comment:
why are we removing localize from these titles?

will these not affect anything in the tradershub?

Reply:
Because for the tracking purpose,we do not require localization.just the English should be passes for data analytics.
It will not affect the tradershub as localization is passed here
<img width="931" alt="Screenshot 2024-05-15 at 9 49 39 AM" src="https://github.com/binary-com/deriv-app/assets/96725617/82c8b280-a33b-4121-b320-2932375b62bb">